### DISCUSSION TITLE
How to retrieve real-time detected images from webcams that YOLOV5 can detect? To be used for further analysis in Open CV and to bring the results back to display in real time together with YOLOV5 ?

### DISCUSSION THREAD

**Discussion by EarthGamingTH** [ultralytics/yolov5#7322] [Q&A]
Hi, I'm doing a project about motorcycle license plate detection and people wearing helmets and not wearing helmets. Including specifying the color of the helmet, I use the YOLOV5 that I have trained on the weight. It is used to detect helmets and license plates, and Open CV is used to identify the license plate number and color of the helmet.
But !!! I have a problem is unable to retrieve real-time detected image from YOLOV5. Please provide some guidance on coding guidance and where to further analyze it in OpenCv.
Thanks, your guidance is highly appreciated.
My code!
![image](https://user-images.githubusercontent.com/102754243/162069405-42751dce-0019-4134-ba8f-ceb935fbee3e.png)
----------------------------------------
Ideas i want
![image](https://user-images.githubusercontent.com/102754243/162071245-5b169d46-0957-41b6-8284-8378abf9ab49.png)

**Comment by glenn-jocher** [comment]
@EarthGamingTH `results.render()` is just a method to overlay detections onto the images in the `results.imgs` array, it returns nothing.
